docs/src/main/sphinx/language/types.rst

```diff
@@ -168,8 +168,9 @@ Example: ``TIME '01:02:03.456'``
 
 Time of day (hour, minute, second, millisecond) with a time zone.
 Values of this type are rendered using the time zone from the value.
+Time zones are expressed as the numeric UTC offset value.
 
-Example: ``TIME '01:02:03.456 America/Los_Angeles'``
+Example: ``TIME '01:02:03.456 -08.00'``
```

Comment:
This is incorrect. It should be `TIME '01:02:03.456 -08:00'`

Same in the other example.

Reply:
Good catch ... note the : instead of . @meggpie

Reply:
Got it! Will make the updates.